examples/bri-3/src/bri/communication/agents/bpiMessagesStorage.agent.ts

```diff
@@ -31,7 +29,8 @@ export class BpiMessageStorageAgent extends PrismaService {
     return bpiMessageModels.map((bpiMessageModel) => {
       return this.mapper.map(
         bpiMessageModel,
-        getType<BpiMessage>(),
+        BpiMessage,
+        BpiMessage,
```

Comment:
why twice?

Reply:
this is because we are mapping from Prisma model to domain model